app/services/metrics.py

```diff
@@ -19,30 +13,23 @@ def get_areas_by_defined_area(area_type, area_id):
         # TODO: Implement service
         return ""
 
-    def get_areas_by_polygon(polygon):
-        # TODO: read raster from STAC
-        raster_cloud_path = "https://staccatalog.blob.core.windows.net/cog-test/Colombia_pp-2015_12_31-pp_2011_2015.tif"
-        out_data = raster_utils.get_raster_values(raster_cloud_path, polygon)
+    def get_areas_by_polygon(
+        polygon: PolygonGeometry, metric_id: str
+    ) -> list[dict[str, float]]:
+
+        first_asset = get_first_item_asset_from_collection(metric_id)
+
+        out_data = raster_utils.get_raster_values(first_asset, polygon)
+
         return out_data
 
     def get_layer_by_defined_area(area_type, area_id):
         # TODO: Implement service
         return ""
 
-    def get_layer_by_polygon(metric_id, polygon):
-
-        collection_metric = next(
-            (
-                collection
-                for collection in collections
-                if collection["id"] == metric_id
-            ),
-            None,
-        )
+    def get_layer_by_polygon(metric_id: str, polygon: PolygonGeometry):
```

Comment:
I just noticed these two functions have the parameters in different order (get_areas_by_polygon and get_layer_by_polygon, please leave metric_id first in both